operator_ui/src/screens/JobProposal/JobProposalScreen.tsx

```diff
@@ -59,8 +59,8 @@ export const REJECT_JOB_PROPOSAL_SPEC_MUTATION = gql`
 `
 
 export const APPROVE_JOB_PROPOSAL_SPEC_MUTATION = gql`
-  mutation ApproveJobProposalSpec($id: ID!) {
-    approveJobProposalSpec(id: $id) {
+  mutation ApproveJobProposalSpec($id: ID!, $force: Boolean) {
+    approveJobProposalSpec(id: $id, force: $force) {
```

Comment:
Is there a place where this gql request actually gets sent a false? I

Reply:
Not currently. It is only used once, and a modal pops up to inform the user that this action will occur